README.md

```diff
@@ -1,17 +1,17 @@
-# spacer_translator
+# Spacer Translator
 
-A project for Flutter International hackathon Hack19
+A project for Flutter International Hackathon Hack19
 
```

Comment:
I guess this isn't an important line, can you please remove it ?

Reply:
@rkishan516 
Removed the line.Any more changes to be done?

Reply:
Nice Work